tests/django_setup.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+from django.db import connection
+
+is_initialized = False
+
+
+def init_db():
+    global is_initialized
+    if is_initialized:
+        return
+    is_initialized = True
+
+    # replace sys.stdout for prompt to delete database
```

Comment:
Django's `django.db.connection.creation.create_test_db(...)` function can prompt for user input, which is problematic if it happens when invoked by the `nose` test runner because by default, `nose` captures STDOUT. This ensures that if Django prompts, the message cannot be obscured by nose's STDOUT capture.